src/app/services/sms/sms.service.ts

```diff
@@ -274,7 +274,7 @@ export const sendVerificationOtp = async (
   }
   const twilioData = await getTwilio(otpData.msgSrvcName, defaultConfig)
 
-  const message = `Use the OTP ${otp} to complete your submission on ${config.app.title}.`
```

Comment:
instead of replacing `config.app.title` with `form.gov.sg` hardcoded, can we use `config.app.appUrl` instead? that way we can accommodate all our non-production environments (staging, UAT, staging-alt). you might need to strip off the `https://` using something like `new URL(config.app.appUrl).host`.

Reply:
also for readability, you can consider using `dedent` to separate the lines instead of newline characters. can see `sms.util.ts` for an example.

Reply:
Thank you @mantariksh !